assets/questions/parens.js

```diff
@@ -68,25 +71,26 @@ def countNumberOfParentheses(s):
       expectedOutput: false
     }],
     buggyOutputTests: [{
-      buggyFunction: 'countNumberOfParentheses',
+      buggyFunction: 'AuxiliaryCode.countNumberOfParentheses',
       messages: [
-        "It looks like you're returning True when you shouldn't.",
+        "This doesn't seem quite right. Check your code and try again.",
         [
           'Are you making sure the parentheses are properly balanced? () ',
           'is balanced, but )( is not.'
         ].join(''),
         [
           "It looks like you're counting the number of parentheses, and if ",
-          "they're equal, returning true. That's not quite correct."
+          "you have the same number of each kind, returning true. That's not ",
+          "quite correct."
         ].join('')
       ]
     }],
     performanceTests: []
   }, {
     instructions: [
       [
-        'Now, make sure your code handles unexpected cases, such as the ',
-        'empty string.'
+        'Now, double-check your code to make sure it handles unexpected cases, ',
+        'such as the empty string.'
```

Comment:
Once you're sure that it does, submit it to see if you're right.

(or similar)